docs/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: The Dafny Language and Software Verification System
+# Feel free to add content and custom Front Matter to this file.
+# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
+
+layout: page
+---
+<script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
+<script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
+<script type="text/x-mathjax-config">
+        MathJax.Hub.Config({tex2jax: {inlineMath: [['$','$'], ["\\(","\\)"]], displayMath: [ ["$$","$$"], ["\\[","\\]"] ]
+        }});
+</script>
+
+<link rel="stylesheet" href="assets/main.css">
+
+**Quick Links:**
+- [Dafny Reference Manual and User Guide](DafnyRef/DafnyRef)
+- [Index to Dafny Resources for Users](toc) ([installation](https://github.com/dafny-lang/dafny/wiki/INSTALL)) ([Dafny for VSCode](https://marketplace.visualstudio.com/items?itemName=dafny-lang.ide-vscode))
```

Comment:
```suggestion
- [Index to Dafny Resources for Users](toc) 
- Install Dafny using the Visual Studio Code [extension](https://marketplace.visualstudio.com/items?itemName=dafny-lang.ide-vscode)
```

Reply:
If we are going to include installation for VSCode here, why not a link to the installation page itself? -- not everyone uses VSCode.

Reply:
Very true, but in my mind, this page doesn't target everyone, but people new to Dafny, who just want to get started. I don't want to overwhelm them with a plethora of options.

Reply:
Right -- so some new person comes to the page -- Linux or Mac user - and wonders how to install. Really? I have to use VSCode? I've never used it before! That's Microsoft! How about a command-line? or emacs?

Reply:
I'm not saying that those user don't exist, but I trust them to find what they're looking for themselves.